app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person_data["name"], person_data["age"])
+                   for person_data in people]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = (
+                Person.people[person["wife"]]
```

Comment:
There is a potential issue here if `person["wife"]` does not exist in `Person.people`. This will raise a `KeyError`. Consider adding a check to ensure the spouse exists before assignment.